hubgrep/frontend_blueprint/templates/components/hosting_service_list/hosting_service_list.html

```diff
@@ -3,7 +3,9 @@
         <ul>
             {% for instance in hosting_services %}
                 <li>
-                    <a class="" href="{{ instance.landingpage_url }}">{{ instance.label }} ({{ instance.type }})</a>
+                    <a class="" href="{{ instance.landingpage_url }}">{{ instance.domain }}</a>
+                    <pre>{{ instance.type }}, {{ instance.repo_count}} repos</pre>
+                    <pre>last updated {{ instance.export_timestamp.strftime('%Y-%m-%d') if instance.export_timestamp else "None"}}</pre> 
```

Comment:
just so that you can see how many repos a hosting service has / how old the state is